### PR TITLE
Automatic language detection bug fix

### PR DESCRIPTION
**Pull request by usajid-aws**
### Description of changes:
Fix for reported bug that selection of multiple languages of the same local under automatic language detection doesn't throw an error. 
### Testing done:
local testing + QA team validation. 
Jenkins adhoc and automated tests builds. 
#### Manual test cases (add more as needed):

- [ ] Join meeting without CallKit
- [ ] Join meeting with CallKit as Incoming
- [ ] Join meeting with CallKit as Outgoing
- [ ] Leave meeting
- [ ] Rejoin meeting
- [ ] See metrics
- [ ] See attendee presence
- [ ] Send audio
- [ ] Receive audio
- [ ] Mute self
- [ ] Unmute self
- [ ] See local mute indicator when muted
- [ ] See remote mute indicator when other is muted
- [ ] Enable and disable local video
- [ ] Enable and disable remote video from remote side
- [ ] Send local video
- [ ] Pause and resume remote video
- [ ] Switch local camera
- [ ] Receive remote screen share
- [x] Start Transcription with Automatic Language Detection

#### Integration validation (required before release)

- [ ] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

### Screenshots, if available:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## [0.19.4] - 2022-04-12
+
+### Fixed
```
Comment (hokyungh) [inline CHANGELOG.md:3]: Looks like this should be in unreleased section?

Comment (usajid-aws) [reply CHANGELOG.md:3]: fixed.

File: AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageOptionsViewController.swift [hunk 2]
```diff
@@ -76,6 +83,16 @@ class AutomaticLanguageOptionsViewController: UIViewController, UITextFieldDeleg
         }
         return selectedIndexPaths
     }
+    
+    private func validateLanguageOptions() -> Bool {
+        for languageGroup in duplicatedLocales.keys {
+            if duplicatedLocales[languageGroup]! > 1 {
```
Comment (georgezy-amzn) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageOptionsViewController.swift:89]: if duplicatedLocales[languageGroup] ?? 0 > 1

Comment (georgezy-amzn) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageOptionsViewController.swift:89]: if duplicatedLocales[languageGroup] ?? 0 > 1

File: AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageOptionsViewController.swift [hunk 3]
```diff
@@ -111,15 +128,26 @@ extension AutomaticLanguageOptionsViewController: UITableViewDelegate, UITableVi
...
     func tableView(_ tableView: UITableView, didDeselectRowAt indexPath: IndexPath) {
         preferredLanguageDropDown.removeAll(where: { languagesDict[languages[indexPath.row]]?.elementsEqual($0) ?? false})
         preferredLanguageSelected = ""
         preferredLanguageTextField.text = preferredLanguageDropDown[0]
+        updateDuplicatedLocales(selectedLanguage: languagesDict[languages[indexPath.row]]!, count: -1)
+    }
+    
+    private func updateDuplicatedLocales(selectedLanguage: String, count: Int) {
+        for languageGroup in languageGroups {
+            if selectedLanguage.contains(languageGroup) {
```
Comment (georgezy-amzn) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageOptionsViewController.swift:147]: why using `contains` instead of `==`?

Comment (georgezy-amzn) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemo/AutomaticLanguageOptionsViewController.swift:147]: why using `contains` instead of `==`?

**Review by georgezy-amzn — APPROVED** [review]

**Review by linsang21 — APPROVED** [review]

**Review by hokyungh — APPROVED** [review]

**Pull request merged**